students/Sigul4/hw1/HW1_Tsigulov.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Double Range Slider</title>
+    <!--Google Fonts-->
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <!--Stylesheet-->
+    <link rel="stylesheet" href="style.css">
+    <script src="HW1_Tsigulov.js"></script>
+</head>
+<body>
+    <div class="wrapper">
+        <div class="values">
+            <span id="range1">
+                0
+            </span>
+            <span> &dash; </span>
+            <span id="range2">
+                100
+            </span>
+        </div>
+        <div class="container">
+            <div class="slider-track"></div>
+            <input type="range" min="0" max="100" value="30" id="slider-1" oninput="slideOne()">
+            <input type="range" min="0" max="100" value="70" id="slider-2" oninput="slideTwo()">
+        </div>
+    </div>
+    
+     <!--Script-->
+     <script >
+        window.onload = function(){
+            slideOne();
+            slideTwo();
+        }
+
+        let sliderOne = document.getElementById("slider-1");
```

Comment:
When you need to get a DOM element use `const` instead `let`

Reply:
Changes let to const